tests/framework/installer/ceph_installer.go

```diff
@@ -798,3 +811,48 @@ spec:
                   path: /run/udev
 `
 }
+
+func (h *CephInstaller) addCleanupPolicy(namespace string) error {
+	cluster, err := h.k8shelper.RookClientset.CephV1().CephClusters(namespace).Get(namespace, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Errorf("failed to get ceph cluster. %+v", err)
+	}
+	cluster.Spec.CleanupPolicy.DeleteDataDirOnHosts = "yes-really-destroy-data"
+	_, err = h.k8shelper.RookClientset.CephV1().CephClusters(namespace).Update(cluster)
+	if err != nil {
+		return fmt.Errorf("failed to add clean up policy to the cluster. %+v", err)
+	}
+	logger.Info("successfully added cleanup policy to the ceph cluster")
+	return nil
+}
+
+func (h *CephInstaller) waitForCleanupJobs(namespace string) error {
+	allRookCephCleanupJobs := func() (done bool, err error) {
+		appLabelSelector := fmt.Sprintf("app=%s", cluster.CleanupAppName)
+		cleanupJobs, err := h.k8shelper.Clientset.BatchV1().Jobs(namespace).List(metav1.ListOptions{LabelSelector: appLabelSelector})
+		if err != nil {
+			return false, fmt.Errorf("failed to get cleanup jobs. %+v", err)
+		}
+		// Clean up jobs might take some time to start
+		if len(cleanupJobs.Items) == 0 {
+			return false, nil
+		}
+		for _, job := range cleanupJobs.Items {
```

Comment:
How about logging the status of the cleanup jobs to help with troubleshooting? And also log if the jobs weren't found yet